packages/flutter/lib/src/material/page_transitions_theme.dart

```diff
@@ -6,6 +6,7 @@ import 'package:flutter/cupertino.dart';
 import 'package:flutter/foundation.dart';
 import 'package:flutter/widgets.dart';
 
+import '../widgets/dual_transition_builder.dart';
```

Comment:
This is a layer violation. You can't import an implementation file from the widget layer directly into the material layer. The widget layer should export the transition builder so that you can just import `package:flutter/widgets.dart` here.

Reply:
Oh I thought that we need hide from here. Sorry for that.